src/pivot-table/components/cells/__tests__/ColumnAdjusterWrapper.test.tsx

```diff
@@ -0,0 +1,56 @@
+import type { stardust } from "@nebula.js/stardust";
+import { render, screen } from "@testing-library/react";
+import React from "react";
+import type { AdjusterCellInfo, DataModel, ExtendedSelections } from "../../../../types/types";
+import TestWithProvider from "../../../__tests__/test-with-providers";
+import ColumnAdjuster from "../ColumnAdjusterWrapper";
+
+describe("<ColumnAdjuster />", () => {
+  let cellInfo: AdjusterCellInfo;
+  let selections: ExtendedSelections;
+  let interactions: stardust.Interactions;
+
+  beforeEach(() => {
+    cellInfo = { canBeResized: true } as AdjusterCellInfo;
+    interactions = { active: true };
+    selections = { isActive: () => false } as ExtendedSelections;
+  });
+
+  const renderAdjuster = () =>
+    render(<ColumnAdjuster cellInfo={cellInfo} columnWidth={100} dataModel={{} as DataModel} isLastColumn={false} />, {
+      wrapper: ({ children }) => (
+        <TestWithProvider selections={selections} interactions={interactions}>
+          {children}
+        </TestWithProvider>
+      ),
+    });
+
+  afterEach(() => jest.clearAllMocks());
+
+  test("should render ColumnAdjuster", () => {
+    renderAdjuster();
+    // TODO: import class string
```

Comment:
oops, 
maybe you are missing this: https://github.com/qlik-oss/nebula-table-utils/blob/0359f594fd3f695e373e5b05046b7822c792516f/src/constants/index.ts#L26

Reply:
good catch! it should be imported from the shared lib

Reply:
done